helpers/monsters.py

```diff
@@ -0,0 +1,25 @@
+from fractions import Fraction
+
+from osrsbox import monsters_api, items_api
+
+monsters = monsters_api.load()
+items = items_api.load()
+
+
+def load_monster_from_api(monster):
+    for x in monsters:
+        if x.name.lower() == monster:
+            return x
+    return False
+
+
+def parse_monster_drops(drops):
+    sorted_drops = []
+    for x_drops in drops:
+        for x_items in items:
+            if x_items.name.lower() == x_drops.name.lower():
```

Comment:
You might consider playing around with letting this search for substrings also. For example in the ge command I have you can search `dragon pickaxe` or `dragon pick` and they both come up. This is accomplished like this:
https://github.com/zedchance/blues_bot.py/blob/c5866d86c443b4b1f271c97d2b24b71291dabebd/helpers/ge.py#L22
If you want to include this change in this PR or to address it after is up to you, but I think its definitely worth messing around with. (Definitely test it because sometimes it can have some unintended consequences.)

Reply:
This seems very helpfull, definetely something il include within the next commit